app/view/about_section.R

```diff
@@ -28,6 +28,31 @@ about_section <- shiny$div(
   )
 )
 
+about_section <- shiny$div(
```

Comment:
1. Your commit message was about removing the link, but actually you are only overwriting the `about_section` object. If you want to remove something, just remove it.
2. But, the link should not be removed! It should be included in the section about Appsilon!!!